0.preprocess-sites/site_processing_config.yaml

```diff
@@ -2,10 +2,10 @@
 master_config:
   step: preprocess-sites
   pipeline: Pooled Cell Painting
-  project_tag: 2018_11_20_Periscope_Calico
+  project_tag: Periscope_Calico
 ---
 core:
-  batch: 20190919_6W_CP074A
+  batch: CP123
```

Comment:
How about BATCH001 instead so it looks like something that could be a real batch name?
We've made sure to include inputs that could be actual user inputs instead of placeholders everywhere else and I'd like to keep that consistent if possible.